### DISCUSSION TITLE
Batch effect corrected data after RunHarmony?

### DISCUSSION THREAD

**Discussion by genomicworks** [satijalab/seurat#7381] [Q&A]
Hi, there, 

Can I get the batch-corrected data after performing RunHarmony command?
Please let me know how to. 

Thank you firstly.

**Comment by bellenger-l** [comment]
Hello,

The Harmony method only gives you "corrected" coordinates not an assay like the RPCA or SCT methods. So you can retrieve cell embeddings in the harmony `DimReduc` slot along with the other reductions (PCA, UMAP,...) : `obj@reductions$harmony@cell.embeddings`

I hope it helps
Best
